src/logger/loggerTaskEx.c

```diff
@@ -232,7 +232,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* Prepare a Sample */
-                struct sample *sample = &g_sample_buffer[bufferIndex];
+                struct sample *sample = g_sample_buffer + bufferIndex;
```

Comment:
We can use either honestly.  C pointer arithmetic does get a little odd looking I agree, I just don't personally like the dereference to reference operations together.  I'll make a point not to change these in future patches if they are written that way.  Good?